.github/workflows/test.yaml

```diff
@@ -0,0 +1,80 @@
+name: build
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Why only master?

Reply:
it was already like this, I can change it if you want. 👍

Reply:
done.